README.md

```diff
@@ -184,9 +184,22 @@ const transactionStatus = xrpClient.getPaymentStatus(transactionHash); // Transa
 **Note:** The example transactionHash may lead to a "Transaction not found." error because the TestNet is regularly reset, or the accessed node may only maintain one
 month of history.  Recent transaction hashes can be found in the XRP Ledger Explorer: https://livenet.xrpl.org/
 
+#### Payment history
+
+An `XRPClient` can return payments to and from an account.
```

Comment:
Maybe "can return a list of payments to and from an account"

I know you're probably copying me ...